Packs/Kafka/Integrations/KafkaV3/KafkaV3.py

```diff
@@ -20,29 +20,28 @@
 
 class KConsumer(Consumer):
     """Empty inheritance class for C-typed class in order to make mocking work."""
-    pass
 
 
 class KProducer(Producer):
     """Empty inheritance class for C-typed class in order to make mocking work."""
-    pass
 
 
 class KafkaCommunicator:
     """Client class to interact with Kafka."""
-    conf_producer: Optional[Dict[str, Any]] = None
-    conf_consumer: Optional[Dict[str, Any]] = None
+    conf_producer: Optional[dict[str, Any]] = None
+    conf_consumer: Optional[dict[str, Any]] = None
     ca_path: Optional[str] = None
     client_cert_path: Optional[str] = None
     client_key_path: Optional[str] = None
     kafka_logger: Optional[logging.Logger] = None
 
     SESSION_TIMEOUT: int = 10000
     REQUESTS_TIMEOUT: float = 10.0
-    POLL_TIMEOUT: float = 1.0
+    POLL_TIMEOUT: float = 5.0
```

Comment:
```suggestion
    POLL_TIMEOUT: float = 5.0 # This timeout is being used in playbooks for _________________
```
Lets help other devs understand this change.